ご確認お願いします。

NorikoAso opened the pull request:
①ロゴの部分のwidthの後の70の左右に半角スペースがあったので、半角スペース部分を省きました。
②widthの部分の半角スペースを無くしてしまったので、入れ直しました。

NorikoAso committed:
①ロゴの部分のwidthの後の70の左右に半角スペースがあったので、半角スペース部分を省きました。
②widthの部分の半角スペースを無くしてしまったので、入れ直しました。

pantomimer commented:
67行目にaltタグが抜けているのでいれてください。
![image](https://user-images.githubusercontent.com/12694276/100469100-01daec00-3119-11eb-8d7d-d7fa889aa08a.png)

NorikoAso committed:
・綺麗になりました。

NorikoAso committed:
・object-fit: coverをcssに追加しました。

NorikoAso commented:
object-fit: coverを追加しました。宜しくお願いします。

pantomimer committed:
見た目的な部分を少し修正

pantomimer committed:
SNSボタンなどを修正

NorikoAso committed:
・経営方針から事業案内まで記載しました。

NorikoAso committed:
テスト中

NorikoAso committed:
・NRI課題の企業情報から、ナレッジインサイトまで作成しましたが、以下自分でググり出来る範囲まで対応して提出しております。
①経営方針から事業案内の中身が見本通りにならず、すみません、自分なりに出来た部分までで提出しております。
②FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
③ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。

NorikoAso commented:
・NRI課題の企業情報から、ナレッジインサイトまで作成しましたが、以下自分でググり出来る範囲まで対応して提出しております。
①経営方針から事業案内の中身が見本通りにならず、すみません、自分なりに出来た部分までで提出しております。
②FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
③ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。

pantomimer commented:
・まず、使っていないCSSを削除しましょう。
すべて使っているならいいですが、全く使用していないCSSを
sentence66のように番号付きで書くのは
何がどうなってるかわからなくなります。

まずやってみて、
どれを消していいかわからない場合は、その旨言ってください。
不要なCSSをカットしたものを私が作ります。

・４列横並びのところは、見本を作りましたので、これを参考にやってみてください。
https://codepen.io/Hacase/pen/gOwMXye

NorikoAso committed:
・不要なCSSを削除完了しました。
・大塚先生の見本を見て、４列の表示をして見ました、見本通りに並びを整列するのが難しいですが、先生のを参考にしながら書いてみました。
昼休みに再度以下をググってみましたが、分からん状態です、すみません。
①FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
②ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。
③h3で設定した、NRI強みの文字がブルーのボックスの中に入ってくれず、、相変わらず苦戦しております。すみません。

NorikoAso commented:
会社の昼休みの時間に以下、早速修正してみました。
・不要なCSSを削除完了しました。
・大塚先生の見本を見て、４列の表示をして見ました、見本通りに並びを整列するのが難しいですが、
　先生のを参考にしながら書いてみました。
再度以下をググってみましたが、分からん状態です、すみません。
①FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
②ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。
③h3で設定した、NRI強みの文字がブルーのボックスの中に入ってくれず、、相変わらず苦戦しております。すみません。

NorikoAso committed:
①FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
②ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。
③h3で設定した、NRI強みの文字がブルーのボックスの中に入ってくれず、、相変わらず苦戦しております。すみません。

NorikoAso commented:
・不要なCSSを削除完了しました。
・大塚先生の見本を見て、４列の表示をして見ました、見本通りに並びを整列するのが難しいですが、
　先生のを参考にしながら書いてみました。
再度以下をググってみましたが、分からん状態です、すみません。
①FollowUsの下線の部分の長さ調整方法がググっても分からず、すみません、出来た部分で出しております。
②ナレッジインサイトから以下はすみませんが、ググっても分からずの状況です。お手数かけますが、教えてください。
③h3で設定した、NRI強みの文字がブルーのボックスの中に入ってくれず、、相変わらず苦戦しております。すみません。

pantomimer commented:
![image](https://user-images.githubusercontent.com/12694276/101597608-45e3c000-3a3a-11eb-8bde-0e1e68ba85bf.png)

pantomimer commented:
![image](https://user-images.githubusercontent.com/12694276/101597750-7c213f80-3a3a-11eb-80f7-74c11ba30177.png)

pantomimer commented:
![image](https://user-images.githubusercontent.com/12694276/101599528-3f0a7c80-3a3d-11eb-84cc-e3a8fd2c6e49.png)

pantomimer commented:
![image](https://user-images.githubusercontent.com/12694276/101599709-8ee94380-3a3d-11eb-92cc-b6a9a9105198.png)
こんな感じです

pantomimer commented:
![image](https://user-images.githubusercontent.com/12694276/101599758-a88a8b00-3a3d-11eb-9670-d13695d8d3a6.png)
見本なのでCSSとかは修正してませんが
こんなふうに４分割になればOKです
企業情報とかのやり方と全く同じです。

pantomimer commented:
ブルーボックスはごまかしで。
（box-wrapperがややこしそうなので
そのまま置いておく）
![image](https://user-images.githubusercontent.com/12694276/101600963-63675880-3a3f-11eb-85f7-7da332c3f0d0.png)

![image](https://user-images.githubusercontent.com/12694276/101601043-82fe8100-3a3f-11eb-8de4-78f98601bad5.png)

![image](https://user-images.githubusercontent.com/12694276/101601075-901b7000-3a3f-11eb-82f9-af9f8975faae.png)

NorikoAso committed:
①ナレッジ・インサイトから、サイトマップまでが横一列表示にどうしてもなってしまっている。

NorikoAso commented:
１時間以上ググり、試行錯誤しましたが、以下事象にもう限界です。
①ナレッジ・インサイトから、サイトマップまでが横一列表示にどうしてもなってしまっている。

NorikoAso committed:
なんとか最後まで完成しました。
①Copyright(c)の部分が免責事項と同じ位置に何度トライしても表示されず苦戦しております。

NorikoAso commented:
なんとか最後まで完成しました。
①Copyright(c)の部分が免責事項と同じ位置に何度トライしても表示されず苦戦しておりますがご確認の程
宜しくお願いします。

pantomimer commented on index.html:
```diff
@@ -53,192 +53,195 @@ <h1>NRIの強み</h1>
-      <li><a href="#">強固な顧客基盤</a></li>
-      <li><a href="#">人材が支えるNRI</a></li>
-      <li><a href="#">業界トップクラスの収益力</a></li>
+  <div class="back" href="#">＜&ensp;前のページに戻る</div>
```
【修正2】
&lt;div 要素にhrefをいれないでください。

pantomimer commented on index.html:
```diff
@@ -53,192 +53,195 @@ <h1>NRIの強み</h1>
-<hr style="border:0;border-top:1px solid #C0C0C0"> 
+  </div>
+ </div>
+ </div>
```
【修正3】
このへんのDIVは何にかかっていますか？
DIVがどこにかかっているかチェックしてください。

pantomimer commented on index.html:
```diff
@@ -53,192 +53,195 @@ <h1>NRIの強み</h1>
+  <li class="footer-item"><a href="#">お問い合わせ</a></li>
+  <li class="footer-item"><a href="#">アクセスマップ</a></li>
 </ul>
+ <p><small>Copyright(c) Nomura Research Institute, Ltd. All rights reserved.</p></small> 
```
【修正4】
&lt;p&gt;&lt;small&gt; ならば &lt;/small&gt;&lt;/p&gt;
にしましょう

pantomimer reviewed (commented):
タグの乱れは、プルリクエスト前にW3cでチェックするようにしてください。
https://validator.w3.org/

【修正１】
![image](https://user-images.githubusercontent.com/12694276/102154736-5cb46780-3ebd-11eb-9b1d-daab99c02a2c.png)

pantomimer commented:
【修正5】
![image](https://user-images.githubusercontent.com/12694276/102155069-16133d00-3ebe-11eb-81de-f937c23d2073.png)

NorikoAso commented:
・修正点で頂きました2,４,5は完了しております。

NorikoAso committed:
修正点で頂きました2,４,5は完了しております。
・copyrightの位置がまだ直り切れていない状況です。
・免責事項の前に｜が表示されている状況です。

pantomimer commented:
【修正6】
問題３に繋がりますが、
divがきちんと入れ子になっていません。
231行目と232行目（footer class="footer"のすぐ上）の&lt;/div&gt; は不要だと思います。

【修正7】
インデントが全体的にバラバラです。
入れ子を意識しないならばインデントを付ける意味がないので、そこは意識して修正してください。

【修正8】
class名にoneとかtwoとかclass-Aなど、意味がない名前をつけるのをやめてください。
何の要素だかわからなくて他の人が困ります。

NorikoAso committed:
修正6,7,8完了しております。

NorikoAso commented:
・修正6,7,8完了しております。

NorikoAso committed:
・企業情報〜事業案内まで綺麗に整えました。
・ナレッジ・インサイト〜サイトマップの部分も綺麗に整えました。

NorikoAso commented:
・企業情報〜事業案内まで綺麗に整えました。
・ナレッジ・インサイト〜サイトマップの部分も綺麗に整えました。

pantomimer commented:
何度もいいますが、
提出前にw3cにかける習慣をつけてください。
![image](https://user-images.githubusercontent.com/12694276/102712604-c68da080-4305-11eb-8235-9f42156cf3dc.png)

pantomimer commented on index.html:
```diff
@@ -6,244 +6,231 @@
+    </div>
+    <div class="sitemap"><a href="#">&gt;&ensp;サイトマップ</a></div>
+  </div>
+</div>
```
【修正9】
&lt;/div&gt;が不要です

pantomimer reviewed (commented):
CSSの中に!important がついているものがあります。
つけてはいけない決まりになっているので、これを外してください。

以上２点が完了したらLGTMとします

NorikoAso committed:
・CSSの中の!important を削除しました。
・【修正9】217行目の</div>が削除しました。
・提出前にW3でNo errors or warnings to show.と表示になりましたので再度提出します。

NorikoAso commented:
・CSSの中の!important を削除しました。
・【修正9】217行目の</div>が削除しました。
・提出前にW3でNo errors or warnings to show.と表示になりましたので再度提出します。

NorikoAso commented:
N高、Html完成しました。
